document/core/syntax/modules.rst

```diff
@@ -236,12 +253,22 @@ The |MELEM| component of a module defines a vector of *element segments* that in
 
 The |EOFFSET| is given by a :ref:`constant <valid-constant>` :ref:`expression <syntax-expr>`.
 
+The |MPELEM| component of a module defines a vector of passive element segments. Each passive element segment only defines its contents.
```

Comment:
Ah, I thought putting them in the same section and index space, just with different records. Not sure I'd particularly fancy introducing new sections... Also, I could imagine there being future features that want to index both kinds uniformly.

Reply:
Oh, I see what you mean. I didn't think of these as being new sections, but I can see why that might be confusing.